core/lib/types/src/contract_verification/api.rs

```diff
@@ -147,6 +147,19 @@ pub struct VerificationIncomingRequest {
     pub is_system: bool,
     #[serde(default)]
     pub force_evmla: bool,
+    #[serde(flatten)]
+    pub evm_specific: VerificationEvmSettings,
+}
+
+/// Settings for EVM verification, used only if
+/// `SourceCodeData` is `SolSingleFile`.
+#[derive(Debug, Clone, Default, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct VerificationEvmSettings {
```

Comment:
The verification request is pretty messy tbh, I think if we will have enough resources, we need to redesign the API and rely on the `standard-json` input only.

Reply:
Can we mark it as resolved to merge this PR? :)